org.cubocore.CorePaint.yml

```diff
@@ -0,0 +1,46 @@
+app-id: org.cubocore.CorePaint
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: corepaint
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-pictures
```

Comment:
Seems to be correctly using portals so no need for `--filesystem` here.

Reply:
But need to save files. Is that also covered by portals?